app/templates/gentelella/guest/event/_ticketing_box.html

```diff
@@ -270,9 +270,9 @@ <h3 class="panel-title">{{ _("Ticket Information") }}</h3>
         var discount_amount = -1;
         var promo_code_status = 0;
         var discount_code = "";
-        {% if code %}
+        {% if discount_code %}
```

Comment:
@niranjan94 okay, but i need some help about how it's going to work? so i need to discuss with someone, if you can help?